admin/dt-prayer-campaigns.php

```diff
@@ -209,12 +243,20 @@ public function main_column() {
                             <input type="hidden" name="campaign_settings_nonce" id="campaign_settings_nonce"
                                     value="<?php echo esc_attr( wp_create_nonce( 'campaign_settings' ) ) ?>"/>
 
+
+                            <?php if ( empty( DT_Porch_Selector::instance()->get_selected_porch_id() ) && empty( DT_Posts::list_posts( "campaigns", [] )["posts"] ) ) : ?>
+                            <h2>Setup Wizard</h2>
+                            <p>
+                                <button type="submit" class="button" name="setup_default" value="ongoing">Setup Landing page for Ongoing Campaign</button>
+                                <button type="submit" class="button" name="setup_default" value="24hour">Setup Landing page Month 24/7 Campaign</button>
+                            </p>
+                            <?php endif; ?>
+
                             <table class="widefat">
                                 <tbody>
                                     <tr>
                                         <td>
-                                            <label
-                                                for="select_porch"><?php echo esc_html( "Select Porch" ) ?></label>
+                                            <label for="select_porch"><?php echo esc_html( "Select Landing Page Type" ) ?></label>
                                         </td>
                                         <td>
                                             <select name="select_porch" id="select_porch"
```

Comment:
Would it make sense to wrap the select porch section in an else statement coming from the if above?

as choosing a porch only makes sense once you have a campaign created...

Also that would fix the 2 differently looking UI elements of the buttons and the porch select being on top of each other.

Reply:
Lets process UI layout and flow separately. I have some ideas.